e2e/process-services-cloud/people-group-cloud-component.e2e.ts

```diff
@@ -178,6 +179,36 @@ xdescribe('People Groups Cloud Component', () => {
                 groupCloudComponentPage.checkGroupIsNotDisplayed(`${groupNoRole.name}`);
             });
 
+            it('[C309996] Should be able to filter groups based on composite roles Activit_Admin', () => {
+                peopleGroupCloudComponentPage.enterGroupRoles(`["${CONSTANTS.ROLES.ACTIVITI_ADMIN}"]`);
+                groupCloudComponentPage.searchGroups(groupActiviti.name);
```

Comment:
can we remove the string interpolation where it's not needed? ` groupCloudComponentPage.checkGroupIsDisplayed(groupActiviti.name);`